firehose.go

```diff
@@ -1,9 +1,11 @@
-package main
+package firehose
```

Comment:
not sure what happened in this file which caused Github to not syntax highlight it?

Reply:
oh weird the tests are def all running not sure what happened there